### DISCUSSION TITLE
Proactively tackle alternatives like Remix.run

### DISCUSSION THREAD

**Discussion by nickluger** [blitz-js/blitz#3002] [General] [answered]
With [Remix](https://remix.run/) there is a new contender on the (still pretty small) React full stack framework market. People looking for Next.js or CRA alternatives will stumble upon both, Blitz and Remix and will inevitably ask themselves: What's the difference? Which suits my project better? Which is future-proof?

This is my SERPs currently (Germany):
![Bildschirmfoto 2021-11-26 um 09 53 10](https://user-images.githubusercontent.com/32647249/143553687-13ae66ea-c036-4540-aee4-405fc6c0bf66.png)

It'd be great if this empty SEO space would be filled by some article guiding people on the differences between these, as others do it, e.g. [Gatsby vs Next](https://www.gatsbyjs.com/features/jamstack/gatsby-vs-nextjs/) and as Blitz already does with the [Next.js comparison.](https://blitzjs.com/docs/why-blitz)

I don't think these feature tables are super-useful, though. Rather some article that emphasizes differences in character, goals, philosophy, DX future plans or similar.

This all, assuming that Blitz would stay on the Next.js train. 😅

> Yeah for sure. It would be easier to build on top of Remix than Next.
https://twitter.com/flybayer/status/1319672449345196033?ref_src=twsrc%5Etfw

It would be really great to hear your thoughts on that @flybayer, here or on https://flightreview.fm/. 

Thanks!

**Comment by nickluger** [comment, marked as answer]
This and the future of Blitz is now being discussed here: https://github.com/blitz-js/blitz/discussions/3075

**Comment by NoelJacob** [comment]
Is currently Blitz.js feature complete with Remix? Or can it be built on top of Remix?
